gym/lib/gym/xcodebuild_fixes/swift_fix.rb

```diff
@@ -27,7 +27,15 @@ def swift_library_fix
           ipa_swift_frameworks.each do |path|
             framework = File.basename(path)
 
-            FileUtils.copy_file("#{Xcode.xcode_path}/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphoneos/#{framework}", File.join(swift_support, framework))
+            begin
+              from = File.join(Xcode.xcode_path, "Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphoneos/#{framework}")
```

Comment:
Instead of XcodeDefault.xctoolchain, I think it should be able to get the toolchain option passed in through gym options. https://github.com/fastlane/fastlane/pull/5864